meta-resin-common/recipes-core/dropbear/dropbear_%.bbappend

```diff
@@ -24,5 +24,6 @@ do_install_append() {
 
     install -d ${D}${sysconfdir}/default
     echo 'DROPBEAR_PORT="22222"' >> ${D}/etc/default/dropbear # Change default dropbear port to 22222
+    echo 'DROPBEAR_RSAKEY_DIR=/var/lib/dropbear' >> ${D}/etc/default/dropber # Change default dropbear host key location
```

Comment:
I don't understand why we moved to var lib dropbear and not bindmount the key file (as we do for other cases).